src/app/models/my-place.ts

```diff
@@ -0,0 +1,17 @@
+export class MyPlace {
+    myPlaceId: number;
+    visited: boolean;
+    userId: number;
+    createdOn: string;
+    googlePlaceId: string;
+    
```

Comment:
Add: 
to top of file: import { User } from "./user";

userId?: User["userId"];
user?: User;

Reply:
Done.